README.md

```diff
@@ -120,6 +144,38 @@ Attempts to play a given track in the queue.  If this track is not in the queue
 **Returns:**
 - retVal `Promise` - A promise that resolves when the track starts playing.  This promise can be rejected so you need to handle any errors with `.catch`
 
+### Search
+#### `search.getResults()`
+Retrieves a [`SearchResults`](#searchresults) object representing the current search results.  Will
+throw an error if the user is not currently searching
+
+**Returns:**
+- retVal `SearchResults` - An object of the structure of a [`SearchResults`](#searchresults) object
+
+#### `search.getSearchText()`
+Retrieves the current string that the user is searching for or the most recent string
+the user searched for.  Basically whatever is in the search input field at the moment.
+
+**Returns:**
+- retVal `String` - The value of the search input field
+
+#### `search.isSearching()`
+Determines if the user is currently searching or not
+
+**Returns:**
+- retVal `Boolean` - True if the user is searching, False otherwise
+
+#### `search.performSearch(text)`
```

Comment:
We should prob allow for passing a callback or returning a promise for when the search completes. It feels shitty to make users set up a `.once` behavior on `change:search-result` when they could have it pre-packaged here.